### PR TITLE
Abstract enemies

### PR DESCRIPTION
**Pull request by jacurick19**

### CODE REVIEW

File: sprint0/sprint0/Enemies/AbstractEnemy.cs [hunk 1]
```diff
@@ -30,14 +30,9 @@ public Enemy(Texture2D texture, Vector2 location, Game1 game)
             health = 50;
             Location = new Rectangle((int)location.X, (int)location.Y, (int)(width * Game1.Scale), (int)(height * Game1.Scale));
             Texture = texture;
-            colorMap = new Dictionary<string, List<Rectangle>>
```
Comment (Hajohnson3m) [inline sprint0/sprint0/Enemies/AbstractEnemy.cs:33]: why no more color map?

Comment (jacurick19) [reply sprint0/sprint0/Enemies/AbstractEnemy.cs:33]: don't need no more color map

File: sprint0/sprint0/Enemies/Goriya.cs [hunk 2]
```diff
@@ -42,119 +30,7 @@ public Goriya(Texture2D texture, Vector2 location, string goriyaColor, Game1 gam
...
-            List<Rectangle> sources = new List<Rectangle>();
-            for (int i = 0; i < numFrames; i++)
-            {
-                sources.Add(new Rectangle(xPos, yPos, width, height));
-                xPos += width + 1;
-            }
-            return sources;
-        }
-
-        public void Draw(SpriteBatch spriteBatch)
```
Comment (Hajohnson3m) [inline sprint0/sprint0/Enemies/Goriya.cs:56]: why did you delete all this- im skeptical

**Review by Hajohnson3m — APPROVED** [review]
